### ISSUE TITLE
doc: document  the includes attribute for cc_import

### ISSUE THREAD

**Issue by yaojingguo** [bazelbuild/bazel#15222]
### Description of the bug:

`cc_import` supports `includes`  attribute. But [cc_import doc](https://docs.bazel.build/versions/5.1.0/be/c-cpp.html#cc_import) does not document it.

<img width="913" alt="截屏2022-04-12 下午10 50 23" src="https://user-images.githubusercontent.com/516386/162990331-c57ad461-85bc-45f1-93fd-60184eeedb0b.png">

And a check of `https://github.com/bazelbuild/bazel/blob/5.1.0/src/main/starlark/builtins_bzl/common/cc/cc_import.bzl` shows that `includes` is supported:

https://github.com/bazelbuild/bazel/blob/815d9e499a32fd4d87525ac0c698c293cf26433d/src/main/starlark/builtins_bzl/common/cc/cc_import.bzl#L231

### What's the simplest, easiest way to reproduce this bug? Please provide a minimal example if possible.

Bazel supports a cc_import with includes attribute. For example:

```
cc_import(
  name = "curl_lib",
  hdrs = glob(["vendor/curl/include/curl/*.h"]),
  includes = [ "vendor/curl/include" ],
  shared_library = "vendor/curl/lib/.libs/libcurl.dylib",
)
```

### Which operating system are you running Bazel on?

macOS 10.15.7

### What is the output of `bazel info release`?

release

**Comment by oscar-garzon** [comment]
Hello! Can I be assign to this issue?

**Comment by yaojingguo** [comment]
Yes, I would like to do it. But I don't have the permission to do the assignment.

@oscar-garzon

**Comment by sgowroji** [comment]
Yes please share the relevant changes in the form of a PR and add this issue in the PR description which fixes it.I Can arrange review for it. Thanks!

**Comment by oscar-garzon** [comment]
Thanks! I'll work on it

**Comment by oscar-garzon** [comment]
I'm having trouble to find the file I've to modified. Is this the file I should modified https://github.com/bazelbuild/bazel/blob/master/src/main/java/com/google/devtools/build/lib/bazel/rules/cpp/BazelCcImportRule.java?

**Comment by aiuto** [comment]
Doc generation is a multistep pipeline. The final .md page is created when we build releases (or nightly).
Just create your PR and someone can review it for the correctness of the text.
